### DISCUSSION TITLE
LiquidMenu use with raspberry pico

### DISCUSSION THREAD

**Discussion by mustafakurnaz** [VasilKalchev/LiquidMenu#85] [Q&A]
hi, i want to use lib on Pico but i encountered with this problem. how can i solve this problem?

LiquidMenu\src/LiquidMenu.h: In instantiation of 'bool LiquidLine::add_variable(T&) [with T = int]':
Documents\Arduino\libraries\LiquidMenu\src/LiquidMenu.h:378:5:   required from 'LiquidLine::LiquidLine(uint8_t, uint8_t, A&, B&) [with A = const char [3]; B = int; uint8_t = unsigned char]'
sketch\ScreenLoader.cpp:56:39:   required from here
\Documents\Arduino\libraries\LiquidMenu\src/LiquidMenu.h:434:37: error: call of overloaded 'recognizeType(int&)' is ambiguous
     DataType varType = recognizeType(variable);
                        ~~~~~~~~~~~~~^~~~~~~~~~
\Documents\Arduino\libraries\LiquidMenu\src/LiquidMenu.h:155:10: note: candidate: DataType recognizeType(bool)
 DataType recognizeType(bool variable); //
          ^~~~~~~~~~~~~
C:\Users\MK\Documents\Arduino\libraries\LiquidMenu\src/LiquidMenu.h:161:10: note: candidate: DataType recognizeType(char)
 DataType recognizeType(char variable);
          ^~~~~~~~~~~~~

**Comment by VasilKalchev** [comment]
Hello, @mustafakurnaz and sorry for the late reply.

I haven't used Pico, can you give me more information on the development environment (e.g. compiler, tools that you are using)?
Are you able to compile [an unmodified example](https://github.com/VaSe7u/LiquidMenu/blob/432c09e1d5cacdfe62fb35d7f4e3180302e5f66c/examples/A_hello_menu/A_hello_menu.ino) for the Pico?
